release-notes/20221103-mss-clamp.md

```diff
@@ -0,0 +1,5 @@
+<!-- markdownlint-disable MD041 -->
+It is now possible to customize the default TCP MSS clamping value set by Submariner also for **non** Globalnet deployments.
+This could be useful in network topologies where MTU issues are seen.
+To force a particular MSS clamping value use the submariner.io/tcp-clamp-mss node annotation on Gateway nodes,
+e.g.  kubectl annotate node < node_name > submariner.io/tcp-clamp-mss=< value >
```

Comment:
```suggestion
It is now possible to customize the TCP MSS clamping value set by Submariner even for non-Globalnet deployments.
This could be useful in network topologies where MTU issues are seen.
To force a particular MSS clamping value, use the `submariner.io/tcp-clamp-mss` node annotation on gateway nodes.
e.g. `kubectl annotate node <node_name> submariner.io/tcp-clamp-mss=<value>`
```

Reply:
This seems like a lot of detail for a RN. I feel like the first sentence should suffice and the rest of it should be part of the docs somewhere.

Reply:
We had the same text while first introducing this with Globalnet: https://submariner.io/community/releases/#other-changes. Agree with Tom that the first sentence should suffice.